metricbeat/module/php_fpm/pool/pool.go

```diff
@@ -13,18 +13,19 @@ import (
 
 // init registers the MetricSet with the central registry.
 func init() {
-	if err := mb.Registry.AddMetricSet("php_fpm", "pool", New, HostParser); err != nil {
-		panic(err)
-	}
+	mb.Registry.MustAddMetricSet("php_fpm", "pool", New,
+		mb.WithHostParser(hostParser),
+		mb.DefaultHost(),
```

Comment:
This should be `mb.DefaultMetricSet()`

Reply:
Oh, what was I thinking about :grimacing: fixed.